src/functionsConfig.js

```diff
@@ -28,6 +28,12 @@ var _setVariable = function(projectId, configId, varPath, val) {
   return runtimeconfig.variables.set(projectId, configId, varPath, val);
 };
 
+var _isReservedNamespace = function(id) {
```

Comment:
`function _isReservedNamespace(id)`?

Making it assigned to a `var` isn't all that necessary (and adds yet another `var` to convert later)

Reply:
I am conflicted about keeping internal style consistent versus doing good JavaScript ... should I change the whole file?